packages/application-shell/src/test-utils.spec.js

```diff
@@ -0,0 +1,258 @@
+// This file tests our test-utils file #inception
+//
+// The TestComponents are expected to work. We run the tests to ensure that
+// the `render` method sets up the tests correctly.
+// This is a bit different from our usual tests, as we are testing our testing
+// tools here instead of actual components.
+import React from 'react';
+import { injectIntl } from 'react-intl';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import { injectFeatureToggle } from '@flopflip/react-broadcast';
+import { Switch, Route } from 'react-router';
+import { render, wait } from './test-utils';
+import { GetApplicationState } from '../../application-shell-connectors';
```

Comment:
This file shows all the different things `render` can influence when testing any component. We now have the majority of things covered 🎈